test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -0,0 +1,229 @@
+Could not find tests for function assemblyproperty
+Could not find tests for function babelfish_get_name_delimiter_pos
+Could not find tests for function babelfish_has_any_privilege
+Could not find tests for function babelfish_remove_delimiter_pair
+Could not find tests for function babelfish_split_object_name
+Could not find tests for function information_schema_tsql._pgtsql_char_max_length
+Could not find tests for function information_schema_tsql._pgtsql_char_octet_length
+Could not find tests for function information_schema_tsql._pgtsql_datetime_precision
+Could not find tests for function information_schema_tsql._pgtsql_numeric_precision
+Could not find tests for function information_schema_tsql._pgtsql_numeric_precision_radix
+Could not find tests for function information_schema_tsql._pgtsql_numeric_scale
+Could not find tests for function information_schema_tsql._pgtsql_truetypid
```

Comment:
These are helper functions starting with `_` and these don't expect relevant tests, can we ignore these functions?